services/iota_wrappers.go

```diff
@@ -352,15 +383,19 @@ func sendChunksToChannel(chunks []models.DataMap, channel *models.ChunkChannel)
 		models.DB.ValidateAndSave(&chunk)
 	}
 
-	channel.EstReadyTime = SetEstimatedReadyTime(Channel[channel.ChannelID], len(chunks))
-	models.DB.ValidateAndSave(channel)
+	if os.Getenv("ENABLE_LAMBDA") == "true" {
+		go batchPowOnLambda(chunks)
+	} else {
```

Comment:
The ProcessUnassignedChunks looks at the number of ready channels and that influences how many chunks it sends.  It also calls a method (in this file, I think) that tracks how long the broker is taking to complete PoW, which determines how often process_unassigned_chunks runs.  These are things we'll probably want to change, before we deploy the lambda changes to prod.  

(This comment isn't in reference to this particular PR, just a general comment)